todo_app/app.py

```diff
@@ -1,11 +1,17 @@
-from flask import Flask
+from flask import Flask, render_template, request
+from werkzeug.utils import redirect
+from todo_app.data.session_items import add_item, get_items
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', index = get_items())
```

Comment:
Calling the list of items "index" is a little odd. Would be nicer to call it "items" or similar so the variable name describes what it is.

"index" is the page name rather than a particular piece of data.